### PR TITLE
Add checks to pre-commit

### PR DESCRIPTION
**Pull request by ggravlingen**
I accidentally committed and pushed the version bump to master (have reverted that). Don't want to do the same mistake again so I'm adding a rule to pre-commit.

On a side note, I use Azure's git repos for work and we can set a rule preventing commits without PRs. Can that be on Github too? I'm not an admin of this repo so I can't check.

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
We can set rules for branches, yes.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -24,6 +24,15 @@ repos:
     hooks:
       - id: isort
         name: isort
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: requirements-txt-fixer
+      - id: check-json
```
Comment (MartinHjelmare) [inline .pre-commit-config.yaml:31]: What do these two hooks do?

Comment (ggravlingen) [reply .pre-commit-config.yaml:31]: requirements-txt-fixer - sorts entries in requirements.txt.

check-json - checks json files for parseable syntax.

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Pull request merged**